src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/containers/DisplayAccordionContainer.tsx

```diff
@@ -0,0 +1,191 @@
+import { withStyles } from '@material-ui/core/styles';
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
+import MuiAccordion from '@material-ui/core/Accordion';
+import AccountBoxIcon from '@material-ui/icons/AccountBox';
+import AccessibilityNewIcon from '@material-ui/icons/AccessibilityNew';
+import AssessmentIcon from '@material-ui/icons/Assessment';
+import DesktopWindowsIcon from '@material-ui/icons/DesktopWindows';
+import FaceIcon from '@material-ui/icons/Face';
+import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
+import MapIcon from '@material-ui/icons/Map';
+import InsertEmoticonIcon from '@material-ui/icons/InsertEmoticon';
+import ApartmentIcon from '@material-ui/icons/Apartment';
+import AccessTimeIcon from '@material-ui/icons/AccessTime';
+import InfoIcon from '@material-ui/icons/Info';
+import { ILayout, ILayoutComponent, ILayoutGroup } from '../layout';
+
+export interface IDisplayAccordionContainer {
+  container: ILayoutGroup;
+  components: (ILayoutComponent | ILayoutGroup)[];
+  // eslint-disable-next-line no-undef
+  renderLayoutComponent: (components: ILayoutComponent | ILayoutGroup, layout: ILayout) => JSX.Element;
+  iconLabel?: string;
+}
+
+const useStyles = makeStyles({
+  root: {
+    width: '120%',
+    padding: 12,
+  },
+  groupTitle: {
+    fontWeight: 500,
+    fontSize: '2rem',
+  },
+  accordionBody: {
+    padding: 30,
+    paddingBottom: 40,
+    display: 'grid',
+    '& .MuiGrid-item': {
+      marginTop: '0.2em',
+    },
+  },
+  spacing: {
+    spacing: 0,
+    justifyContent: 'start',
+    gridGap: '12px 24px',
+  },
+  icon: {
+    flexBasis: '10%',
+    flexShrink: 0,
+  },
+  hover: {
+    '&:hover': {
+      backgroundColor: AltinnAppTheme.altinnPalette.primary.greyLight,
+    },
+  },
+  hoverError: {
+    '&:hover': {
+      backgroundColor: AltinnAppTheme.altinnPalette.primary.redLight,
+    },
+  },
+});
+
+const AccordionSummary = withStyles((theme) => ({
+  root: {
+    boxShadow: theme.shadows[1],
+  },
+}))(MuiAccordionSummary);
+
+const Accordion = withStyles((theme) => ({
+  root: {
+    [theme.breakpoints.up('md')]: {
+      '&$expanded': {
+        margin: '0 -3.5%',
+      },
+    },
+  },
+  expanded: {},
+}))(MuiAccordion);
+
+export function DisplayAccordionContainer(props: IDisplayAccordionContainer) {
+  const GetHiddenSelector = makeGetHidden();
+  const hidden: boolean = useSelector((state: IRuntimeState) => GetHiddenSelector(state, { id: props.container.id }));
+  const classes = useStyles();
+  const title = useSelector((state: IRuntimeState) => {
+    const titleKey = props.container.textResourceBindings?.title;
+    if (titleKey) {
+      return getTextFromAppOrDefault(titleKey, state.textResources.resources, state.language.language, [], true);
+    }
+    return undefined;
+  });
+  const layout = useSelector((state: IRuntimeState) => state.formLayout.layouts[state.formLayout.uiConfig.currentView]);
+  const hasErrors = false;
```

Comment:
- [ ] This boolean is only read and never changed. Should it be a local state using `const [hasError, setHasError] = useState(false)` ?